assets/js/modules/analytics/components/settings/SettingsForm.js

```diff
@@ -22,20 +22,19 @@
 import {
 	AccountSelect,
 	AnonymizeIPSwitch,
-	ErrorNotice,
 	ExistingTagNotice,
 	ProfileSelect,
 	PropertySelect,
 	TrackingExclusionSwitches,
 	UseSnippetSwitch,
 	ProfileNameTextField,
 } from '../common';
-
+import StoreErrorNotice from '../../../../components/StoreErrorNotice';
+import { STORE_NAME } from '../../datastore/constants';
```

Comment:
Please keep a blank line between any imports and a function/component definition.
```suggestion
import { STORE_NAME } from '../../datastore/constants';

```